modules/form-field/react/stories/stories_TextInput.tsx

```diff
@@ -1,16 +1,52 @@
 /// <reference path="../../../../typings.d.ts" />
 import * as React from 'react';
 import {storiesOf} from '@storybook/react';
+import {action} from '@storybook/addon-actions';
 import withReadme from 'storybook-readme/with-readme';
 import {InputProviderDecorator, controlComponent} from '../../../../utils/storybook';
 
 import {TextInput} from '../../../text-input/react/index';
-import FormField from '../index';
+import {MenuItem} from '../../../menu/react/index';
+import FormField, {Combobox} from '../index';
 import README from '../../../text-input/react/README.md';
 
 const hintText = 'Helpful text goes here.';
 const hintId = 'error-desc-id';
 
+class Autocomplete extends React.Component<{}, {currentText: string}> {
+  state = {
+    currentText: '',
+  };
+
+  autocompleteCallback = (event: React.ChangeEvent<HTMLInputElement>): void => {
+    console.log('callback')
+    this.setState({ currentText: event.target.value });
+  };
+
+  render() {
+    const autocompleteResult = (textModifier: string) => (
+      <MenuItem onClick={action(`Went to link Result ${textModifier}`)}>
+        Result <span>num<span>ber</span></span> {textModifier}
+      </MenuItem>
+    );
+    return (
+      <Combobox
```

Comment:
That make sense, in that case should it live with text input. or be all on its own in `modules/combobox/react/...`

Reply:
I think it should be its own component 👍 Honestly it could render the `TextInput` itself, but I guess this does provide a bit more flexability in terms of styling for the input.

Reply:
Yep I agree 👍 will do